src/log.h

```diff
@@ -25,32 +25,39 @@
 
 namespace rhost {
     namespace log {
+        enum class log_verbosity {
```

Comment:
It is a valid question and usually verbosity matches level (i.e. errors are visible in minimal and so on). However, in our case verbosity more relates to privacy/security. So 'trace' may be OK as 'minimal' but error may not be logged at minimal if it contains PI or secure content. 

We may consider different name or even means to file error message without associated data